modules/apps/depot/depot-test/src/testFunctional/tests/MyDMTest.testcase

```diff
@@ -0,0 +1,64 @@
+@component-name = "portal-document-management"
+definition {
+
+	property testray.main.component.name = "Document Management";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		task ("Given DM Portlet is added to the page") {
```

Comment:
Again, I really liked that you added the given already in the setup part, but so that we don't have any steps established outside of the Given-When-Then pattern, move these steps that are outside into the task to inside, since they are part of the Given process